docs/get_started.md

```diff
@@ -0,0 +1,38 @@
+# Get Started
```

Comment:
This document should be added to the website.
The rule is if you're modifying Open Match then /docs/ otherwise /site/...